src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {Animated, Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
-import MemoIc_Checklist from '../../../assets/icons/Ic_Checklist';
+import IconChecklist from '../../../assets/icons/Ic_Checklist';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
 import {COLORS} from '../../../utils/theme';
 
-const ItemUser = ({photo, username, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
   const [isSelect, setIsSelect] = React.useState(false);
   const followIconFadeAnimation = React.useRef(new Animated.Value(0)).current;
+  const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` and the renaming of `IconChecklist` are good changes for type checking and readability. The addition of the `isAnon` prop and its use in defining `anonColorCondition` is a logical way to handle different colors based on whether the user is anonymous or not.
```diff
+ import PropTypes from 'prop-types';
+ import IconChecklist from '../../../assets/icons/Ic_Checklist';
+ const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->